### ISSUE TITLE
Duplicates or a lot of times even quadruplicates or more on Transifex

### ISSUE THREAD

**Issue by kiskunk** [Project-OSRM/osrm-text-instructions#273]
The 286 strings could be around 100-150-ish if the redundant duplicates or sometimes quadruplicates could be removed. For example "Take the ramp on the left" is there six times.

**Comment by 1ec5** [comment]
Yes, it is quite a lot. Unfortunately, we do need some of this flexibility in order to accommodate languages that have different words for on-ramps versus off-ramps (#272) and the presence of “onto” or “towards” prepositional phrases.

When we explored localization formats in #19, we had only a few formats to choose from, not all of them supported well by Transifex. I do wonder whether we would’ve made the same decision had [Mozilla’s Fluent](https://projectfluent.org/) been available at the time.
